modules/failing-to-load-the-kernel-driver.adoc

```diff
@@ -11,28 +11,29 @@ Consider the following Collector log:
 
 [source,terminal]
 ----
-[INFO    2022/10/13 14:25:13] Hostname: 'hostname'
+[INFO    2023/05/13 14:25:13] Hostname: 'hostname'
 [...]
-[INFO    2022/10/13 14:25:13] Successfully downloaded and decompressed /module/collector.ko
-[INFO    2022/10/13 14:25:13]
-[INFO    2022/10/13 14:25:13] This product uses kernel module and ebpf subcomponents licensed under the GNU
-[INFO    2022/10/13 14:25:13] GENERAL PURPOSE LICENSE Version 2 outlined in the /kernel-modules/LICENSE file.
-[INFO    2022/10/13 14:25:13] Source code for the kernel module and ebpf subcomponents is available upon
-[INFO    2022/10/13 14:25:13] request by contacting support@stackrox.com.
-[INFO    2022/10/13 14:25:13]
+[INFO    2023/05/13 14:25:13] Successfully downloaded and decompressed /module/collector.o
+[INFO    2023/05/13 14:25:13]
+[INFO    2023/05/13 14:25:13] This product uses kernel module and ebpf subcomponents licensed under the GNU
```

Comment:
Does "kernel module" language stay if kernel module is being removed? (Again, logs are probably outside scope but I thought I would ask.)

Reply:
Good spot! I must have missed this in the original collector PRs. I'll remove it from here and then also remove it from the logging before release.

Reply:
oh, in fact this logging has already changed to reflect this - I'll make the necessary changes here as well